translations/ui-erm-usage/en.json

```diff
@@ -106,7 +106,7 @@
   "reportOverview.month.12": "Dec",
   "reportOverview.report": "Report",
 
-  "reportOverview.confirmDelete": "Delete non-counter report?",
+  "reportOverview.confirmDelete": "Delete report?",
```

Comment:
I'm not sure if just changing the translated value will trigger new translations for other languages. But even if they are triggered the translated values wont match the new english text for a while.
I think its best to use a new key here, so that no incorrect translation is used.